src/carbon-calculation/interfaces/EC3Configurations.ts

```diff
@@ -157,11 +197,39 @@ export interface EC3ConfigurationSingle {
 }
 
 /**
- * Properties of the EC3 Configuration to be updated.
+ * Properties of the EC3 Configuration to be updated. Part of Carbon Calculation EC3 API report schema.
+ * @export
+ * @interface EC3ReportConfigurationUpdate
+ */
+export interface EC3ReportConfigurationUpdate extends EC3ConfigurationUpdate {
+  /**
+   * List of EC3 Configuration Labels.
+   * @type {Array<EC3ConfigurationLabel>}
+   * @memberof EC3ConfigurationUpdate
+   */
+  labels: EC3ReportConfigurationLabel[];
+}
+
+/**
+ * Properties of the EC3 Configuration to be updated. Part of Carbon Calculation EC3 API extraction schema.
+ * @export
+ * @interface EC3ReportConfigurationUpdate
+ */
+export interface EC3ExtractionConfigurationUpdate extends EC3ConfigurationUpdate {
+  /**
+   * List of EC3 Configuration Labels.
+   * @type {Array<EC3ConfigurationLabel>}
+   * @memberof EC3ConfigurationUpdate
+   */
+  labels: EC3ExtractionConfigurationLabel[];
+}
+
+/**
+ * Properties of the EC3 Configuration to be updated. Contains properties shared by both schemas.
  * @export
- * @interface EC3ConfigurationUpdate
+ * @interface EC3ReportConfigurationUpdate
  */
-export interface EC3ConfigurationUpdate {
+interface EC3ConfigurationUpdate {
```

Comment:
Why is this no longer exported?

Reply:
I used this interface to move out properties shared by both schemas, should I put all properties in both schemas instead?

Reply:
I see no problem moving out the common properties to a common interface. I just see no reason why it's not exported. What you had before was fine.

Reply:
I didn't export it since I didn't see any potential use for it. I thought, that it would only confuse the users.

Reply:
Sounds good. I can see how this keeps the API surface minimal. I would put a `Common` suffix at the end of that interface though.